mapper/cgralib.hpp

```diff
@@ -1,11 +1,11 @@
 #ifndef CGRALIB_HPP_
 #define CGRALIB_HPP_
 
-#include "context.hpp"
+#include "coreir.h"
 
 using namespace CoreIR;
 
-Namespace* getcgralib(Context* c) {
+Namespace* CoreIRLoadLibrary_cgra(Context* c) {
```

Comment:
This library definition doesn't follow the convention we outlined [here](https://github.com/rdaly525/coreir/blob/e33bebe1bdb3463f481a392b42ef314494dd7d25/README.md#libraries).

Don't think it should affect the flow if it's only being used internally by the mapper with C++. If we want to merge before updating we should open an issue about updating this to the library convention.